python/orca/test/bigdl/orca/learn/ray/tf/test_tf_ray_estimator.py

```diff
@@ -476,7 +476,11 @@ def model_creator(config):
                     label_cols=["label"])
         trainer.evaluate(train_data_shard, batch_size=4, num_steps=25,
                          feature_cols=["user", "item"], label_cols=["label"])
-        trainer.predict(train_data_shard, feature_cols=["user", "item"]).collect()
+        
+        result = trainer.predict(train_data_shard, feature_cols=["user", "item"])
+        predictions = result.collect()[0]
+        import pandas as pd
+        assert isinstance(predictions, pd.DataFrame), "predict should return a pandas dataframe"
```

Comment:
same as above

Reply:
This file is tested in `pytest-horovod-tf2`, do we need to spilt this ut out? In the horovod test, some ut(`test_string_input`) requires tf==2.3.0.

Reply:
For examples without horovod, all tests should not put in ray test.